src/components/arbeidsoppgaver/ArbeidsoppgaveCard.tsx

```diff
@@ -25,20 +25,36 @@ import { Arbeidsoppgave } from "../../types/oppfolgingsplan";
 
 interface Props {
   innloggetFnr: string;
+  arbeidstakerFnr?: string;
   arbeidsoppgave: Arbeidsoppgave;
   readonly?: boolean;
 }
 
 export const ArbeidsoppgaveCard = ({
   innloggetFnr,
+  arbeidstakerFnr,
   arbeidsoppgave,
   readonly = true,
 }: Props) => {
   const { isAudienceSykmeldt } = useAudience();
   const type = arbeidsoppgave.gjennomfoering?.kanGjennomfoeres;
   const [editererArbeidsoppgave, setEditererArbeidsoppgave] = useState(false);
   const aktoerHarOpprettetElement =
```

Comment:
Her har du den jeg lagde for ca samme case. Mulig vi kan lage en slags util funksjon, eller hook eller noe: https://github.com/navikt/oppfolgingsplan-frontend/blob/f799717d6a817ad90a199608a635d646738c2854/src/components/blocks/dialog/Dialog.tsx#L41